plugin/ipythonPlugins/src/dist/ipython/ipython.js

```diff
@@ -367,7 +370,7 @@ define(function(require, exports, module) {
           }
           if (finalStuff === undefined) {            
             finalStuff = evaluation;
-            bkHelper.timeout(doFinish,150);
+            bkHelper.timeout(doFinish,250);
```

Comment:
this change should be applied to python3.js and all the other similar plugins (each place with the same delay).